app/views/transport.scala.html

```diff
@@ -15,56 +15,126 @@
+                    checked = form("modeOfTransport").value.contains(Transport.ModesOfTransport.Road)
+                ),
+                RadioItem(
+                    value = Some(Transport.ModesOfTransport.Air),
+                    content = Text(messages(s"transport.modeOfTransport.$Air")),
+                    checked = form("modeOfTransport").value.contains(Transport.ModesOfTransport.Air)
+                ),
+                RadioItem(
+                    value = Some(Transport.ModesOfTransport.PostalOrMail),
+                    content = Text(messages(s"transport.modeOfTransport.$PostalOrMail")),
+                    checked = form("modeOfTransport").value.contains(Transport.ModesOfTransport.PostalOrMail)
+                ),
+                RadioItem(
+                    value = Some(Transport.ModesOfTransport.FixedInstallations),
+                    content = Text(messages(s"transport.modeOfTransport.$FixedInstallations")),
+                    checked = form("modeOfTransport").value.contains(Transport.ModesOfTransport.FixedInstallations)
+                ),
+                RadioItem(
+                    value = Some(Transport.ModesOfTransport.InlandWaterway),
+                    content = Text(messages(s"transport.modeOfTransport.$InlandWaterway")),
+                    checked = form("modeOfTransport").value.contains(Transport.ModesOfTransport.InlandWaterway)
+                ),
+                RadioItem(
+                    value = Some(Transport.ModesOfTransport.Other),
+                    content = Text(messages(s"transport.modeOfTransport.$Other")),
+                    checked = form("modeOfTransport").value.contains(Transport.ModesOfTransport.Other)
+                )
+            ),
+            errorMessage = form("modeOfTransport").error.map(err => ErrorMessage(content = Text(messages(err.message, err.args:_*))))
+        ))
 
-        @components.input_text(
-            field = form("nationality"),
-            label = messages("transport.nationality.question"),
-            labelClass = Some("bold-small"),
-            hint = Some(messages("transport.nationality.hint"))
-        )
+        @govukInput(Input(
+            id = "transportId",
+            name = "transportId",
+            classes = "govuk-!-width-one-third",
+            label = Label(
+              content = Text(messages("transport.transportId.question")),
+              classes = "govuk-label--m"
+            ),
+            hint = Some(Hint(
+              content = Text(messages("transport.transportId.hint"))
+            )),
+            errorMessage =
+              form("transportId").error.map(err => ErrorMessage(content = Text(messages(err.message, err.args:_*)))
+            )
+        ))
 
-        @components.submit_button(messages("site.continue"))
+        @govukInput(Input(
+            id = "nationality",
+            name = "nationality",
+            classes = "govuk-input--width-2",
```

Comment:
Should the two input fields not have the same class(Style)?

Reply:
I believe they should not. Nationality is much shorter than the transport id.

Reply:
Fair enough.  They used to be the same size but not on the latest prototype.